core/src/main/java/org/frankframework/senders/MailSender.java

```diff
@@ -109,6 +109,7 @@ public class MailSender extends MailSenderBase {
 
 	private Properties properties = new Properties();
 	private Session session = null;
+	private final boolean useSsl = true;
```

Comment:
Zou je voor dit veld een `setter` kunnen toevoegen zodat de Frank developer controle krijgt over dit attribuut vanuit de configuratie XML.

Daarnaast zou ik het standaard op `false` zetten zodat iedereen die tot nu toe een `mailSender` heeft geimplementeerd niet overal `useSssl="false"` hoeft aan te brengen.

Reply:
Jazeker, de change is gepusht.
Gezien de inhoud van het Ticket had ik het veld op true gezet

Reply:
Ik snap denk ik waar de miscommunicatie is ontstaan, in het ticket refereer ik naar 'nieuw attribuut' met waarde `true` inderdaad. 

Hiermee bedoelde ik het beschikbaar stellen van nieuw attribuut in de configuratie XML voor de Frank developer en het kunnen activeren van SSL wanneer de Frank developer dit attribuut voorziet met de waarde `true`. Dat kan je realiseren door in de class een `set` methode voor het desbetreffende attribuut toe te voegen.